src/test/fuzz/coinscache_sim.cpp

```diff
@@ -148,8 +146,10 @@ class CoinsViewBottom final : public CCoinsView
 public:
     std::optional<Coin> GetCoin(const COutPoint& outpoint) const final
     {
-        // TODO GetCoin shouldn't return spent coins
-        if (auto it = m_data.find(outpoint); it != m_data.end()) return it->second;
+        if (auto it = m_data.find(outpoint); it != m_data.end()) {
```

Comment:
nit: for consistency with other overrides:
```suggestion
        if (auto it{m_data.find(outpoint)}; it != m_data.end()) {
```